internal/http/types/generic.go

```diff
@@ -16,14 +16,19 @@ type Response struct {
 }
 
 type Pagination struct {
-	Page int64 `json:"page"`
-	Size int64 `json:"size"`
+	Page      int64  `json:"page"`
```

Comment:
Is this still used?
I see that the rules API is using it, but AFAICT it is no longer parsed from the URL query params.

Reply:
good catch, i missed that 

should be fixed now